apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx

```diff
@@ -28,29 +33,34 @@ export default function RubricsContainer({
             learningGoalEvidenceLevelsAttributes: [
               {
                 teacherDescription: '',
-                understanding: 0,
+                understanding: RubricUnderstandingLevels.NONE,
                 aiPrompt: '',
               },
               {
                 teacherDescription: '',
-                understanding: 1,
+                understanding: RubricUnderstandingLevels.LIMITED,
                 aiPrompt: '',
               },
               {
                 teacherDescription: '',
-                understanding: 2,
+                understanding: RubricUnderstandingLevels.CONVINCING,
                 aiPrompt: '',
               },
               {
                 teacherDescription: '',
-                understanding: 3,
+                understanding: RubricUnderstandingLevels.EXTENSIVE,
                 aiPrompt: '',
               },
             ],
           },
         ]
   );
 
+  const [saveNotificationText, setSaveNotificationText] = useState('');
+
+  const SAVING_TEXT = 'Saving...';
```

Comment:
I feel like this is probably true, but I'm used to looking for this. Do we not localize the levelbuilder? It makes sense not to waste translation effort on it, but I was curious.

Reply:
Nope, we do not. In fact, we don't let levelbuilders change the language of the site (we had a couple of instances of translated content being saved as the source of truth and it was annoying)